### ISSUE TITLE
Detection with Trained model

### ISSUE THREAD

**Issue by sourav164** [eriklindernoren/PyTorch-YOLOv3#196]
I am new in deep learning. I have trained the yolo_v3 model with my custom dataset and got the checkpoint weights by running the following code.

    python3 train.py --model_def config/yolov3-custom.cfg --data_config config/custom.data --epochs 35 --batch_size 6 

I need to apply the model to identify objects in a new dataset using the trained model. How can I use the checkpoint weights? Is that something like -

    detect.py --image_folder data/new_data --model_def config/yolov3-custom.cfg  --checkpoint_model checkpoints/yolov3_ckpt_10.pth

**Comment by blueclowd** [comment]
Have same questions: 
Below is my command but doesn't work (no detection):
 python3 detect.py --image_folder data/custom/images/ --model_def config/yolov3-custom.cfg --class_path data/custom/classes.names --checkpoint_model checkpoints/yolov3_ckpt_29.pth --conf_thres 0.1

**Comment by Invokar** [comment]
@sourav164 @blueclowd 
As it is said in readme. It's ok when use `python detect.py`
The only thing you should be sure is to change the `default` argument to your image path
```python
parser.add_argument("--image_folder", type=str, default="data/samples/", help="path to dataset")
```

**Comment by sourav164** [comment]
@Invokar 
I had some new image (inside data/new_data folder) where I wanted to apply the newly trained model.  Thus, I provided its path

    --image_folder data/new_data

I think I provided the right path, still getting no result using the newly trained model.

**Comment by nanhui69** [comment]
@sourav164  do you solve that problem?  i meet the same problem when i retrain the yolo project,and i define my custom image, like python detect.py --image_folder data/new_data  --checkpoint_model checkpoints/yolov3_ckpt_2.pth，but the detection is none,，so how to do ？

**Comment by ChalsonLee** [comment]
> @sourav164 do you solve that problem? i meet the same problem when i retrain the yolo project,and i define my custom image, like python detect.py --image_folder data/new_data --checkpoint_model checkpoints/yolov3_ckpt_2.pth，but the detection is none,，so how to do ？

I think that's probably because you only trained two epochs and didn't have enough precision to detect them. If there is only a label but no rectangle, it is probably because the predicted rectangle is too small, less than 1 pixel, to be drawn

**Comment by Flova** [comment]
Is this issue still relevant/occurring?

I closed this issue due to inactivity. Feel free to reopen for further discussion.
